### ISSUE TITLE
bot closes draft PRs and says to re-open them but there's no option to re-open them

### ISSUE THREAD

**Issue by omajid** [dotnet/runtime#51867]
I had a PR open in draft mode: https://github.com/dotnet/runtime/pull/50292

`@msftbot` closed it with a message saying that I should re-open it manually.

But this option is not available to me.

![image](https://user-images.githubusercontent.com/174451/116109605-f3bad880-a682-11eb-8459-f53732549c95.png)

Would it be possible for the bot to stop closing PRs like this?

**Comment by ViktorHofer** [comment]
cc @danmoseley

**Comment by ViktorHofer** [comment]
I can't reopen the PR either (and I'm an admin) 🗡️

**Comment by jaredpar** [comment]
Presently you can't open because @omajid has already created a new PR from that branch. GitHub won't let you open two PRs from the same fork + branch. Hence the Reopen is disabled. 

Why this couldn't be done originally isn't clear to me though.

**Comment by akoeplinger** [comment]
Users without write permissions can't reopen a PR/issue if it was closed by someone else, that's a long standing GitHub limitation: https://github.com/isaacs/github/issues/583

I think we should reword the message to say "please open a new PR" or something like that.

**Comment by jaredpar** [comment]
Oh that's interesting. I wasn't aware of that limitation. 

In that case I think we should reword it to basically say something along the lines of: 

> Please let us know you're interested in continuing working on this and we'll reopen the pull request. 

I'd like to avoid pushing people to open a new PR because that can lose valuable feedback comments.

**Comment by danmoseley** [comment]
OK, changed to:

Draft Pull Request was automatically closed for inactivity. Please [let us know](https://github.com/dotnet/runtime/blob/main/docs/area-owners.md) if you’d like to reopen it.

**Comment by ViktorHofer** [comment]
Closing as the bot message got rephrased.
